LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/LoggerConfiguration.cs

```diff
@@ -32,5 +32,26 @@ public class LoggerConfiguration
 
         // Gets/sets udp port to send logs
         public int LogToUdpPort { get; set; } = 6000;
+
+        public static int InitLogLevel(string logLevel)
+        {
+            int logLevelInt = 4;
+            logLevel = logLevel.ToUpper();
```

Comment:
You can avoid the allocation of the string .ToUpper by comparing strings ignoring case:
`string.Equals(str1, str2, StringComparison. InvariantCultureIgnoreCase)`

Reply:
Done! More elegant code indeed.